addons/website/views/snippets/s_text_cover.xml

```diff
@@ -5,7 +5,7 @@
     <section class="s_text_cover">
         <div class="container-fluid">
             <div class="row">
-                <div class="o_colored_level o_cc o_cc_1 col-lg-5 pt160 pb160 px-5">
+                <div class="o_colored_level o_cc o_cc4 col-lg-5 pt160 pb160 px-5">
```

Comment:
@anso-odoo Should not this target the version where we introduced the error ? :wink: Also if you can mention the commit which created the error in your commit message, that would be nice too, thanks :pray:

Reply:
Hello @qsm-odoo  ! Makes sense, thanks. I changed the base of the branch to 16.3 and added the mention :)

Reply:
> Should not this target the version where we introduced the error ? 

I'm not sure that's a FIX, asaik it's an "improvement" suggested by fp/pde.
Just a note tho, feel free to ignore my message if there are tech. reasons for targeting prev. version

Reply:
@qsm-odoo , I agree with Stefano on this one.
It's not a big deal, but we might as well move this to 16.4 to make sure we don't change customer's website without them upgrading to a new version.
Obviously, in this case, both approaches works.

Reply:
> It's not a big deal, but we might as well move this to 16.4 to make sure we don't change customer's website without them upgrading to a new version.

@pierre-pde I don't know what you mean, we are talking about fixing a snippet structure. How would that change customers' websites?

> I'm not sure that's a FIX, asaik it's an "improvement" suggested by fp/pde.
> Just a note tho, feel free to ignore my message if there are tech. reasons for targeting prev. version

@stefanorigano I guess there is no real issue indeed at the moment... for now. But o_cc comes with CSS rules, it is meant to be used with a color class (o_ccX) which come with its own CSS rules. When it's a recent version (saas-16.3), I prefer in general fixing the arch of snippets when we can (here we can: just adding css class) to prevent accumulating bugs in upgraded websites. Let people drop a "standalone" o_cc column in 16.3 "because there is no bug", maybe in 17.0 we add the rule `.o_cc a { color: blue }` (thinking it will be overridden by the rule `.o_ccX a { color: red; }` and their website will be broken post upgrade.

Basically it's the same as writing a wrong bootstrap which is visually not broken: maybe it will be, why not preventing users to drop the *technically* broken one? 

Anyways, it's of course a detail. Just wanted to explain my thinking :wink:

Reply:
@qsm-odoo , maybe I misunderstood. I thought this PR was aiming at selecting Preset 4 instead of preset 1.
The idea being that preset 4 has dark bg which should look better.

looking at the line diff, I understand that o_cc_1 was maybe a typo anyway? 
Then of course we should aim 16.3 :D

Reply:
Hello @qsm-odoo, just rebased. If I understood the comments correctly, we are good to go ? Ty !